lib/config.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+
+var config;
+
+config = {
+	bandwidth : {
+		userId : process.env.BW_USER_ID,
+		token  : process.env.BW_TOKEN,
+		secret : process.env.BW_SECRET
+	},
+
+	isSet : function (prop) {
```

Comment:
I'm not sure that I understand the purpose of this pattern. It seems to me that it would be simpler, more performant, and less prone to unintented side effects to simply use normal JS variable resolution:
```
var config = require("config");
if (!config.bandwidth.userId) {
    throw new Error();
}
```